packages/itwin/ec3-widget/public/locales/en/ec3Widget.json

```diff
@@ -0,0 +1,35 @@
+{
+  "ec3Widget": {
+    "stepOneTitle": "Create a template",
+    "stepTwoTitle": "Add Assembly(s)",
+    "stepThreeTitle": "Send report to EC3",
```

Comment:
Was this string in the UX mockup? Everywhere else `Report` R is uppercase.

Reply:
used sentence case everywhere

Reply:
Isn't Reports a proper noun though? It's not just a report but an iTwin Platform Insights and Reporting _Report_

Reply:
capitalized report everywhere